meta/documents/user_guide/en/initial_setup.md

```diff
@@ -181,9 +181,31 @@ If Wayfair's shipping services are to be used, the Wayfair plugin's ASN settings
 
 16. Click the `Save` button ![save button](../../../images/common/button_save.png).
 
-## 8. Performing the first inventory synchronization
-Once everything has been set up, the it is time to start listing items for sale on Wayfair.
+## 8. First inventory synchronization
+Once everything has been set up, the it is time for the Wayfair plugin to start listing items for sale on Wayfair.
 
-Finalize the setup by initiating a full inventory synchronization on the [Full Inventory](settings_guide.md#full-inventory-page) page of the [Wayfair Market Settings](settings_guide.md).
+### Version 1.1.4 and later
 
-**After this first inventory synchronization, the Wayfair plugin will periodically send inventory updates to Wayfair, without any further manual activations.**
+As of version 1.1.4, the Wayfair plugin's Inventory Synchronization process is fully automated and cannot be started by the Plentymarkets user. After completing of all earlier steps in this guide, it is important to ensure that the Wayfair plugin is able perform Inventory Synchronization:

```

Comment:
```suggestion
As of version 1.1.4, the Wayfair plugin's Inventory Synchronization process is fully automated and cannot be started by the Plentymarkets user. After completing all earlier steps in this guide, it is important to ensure that the Wayfair plugin is able perform Inventory Synchronization:
```
Alternatively it could read as `After completion of`